### PR TITLE
Minimize_Amplitude_After_K_Removals.py

### PR DESCRIPTION
**Pull request by kushal13112001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

I have uploaded the file for Minimize Amplitude After K Removals.py #4438 

### Checklist

- [ ✔️ ] I've read the contribution guidelines.
- [ ✔️ ] I've checked the issue list before deciding what to submit.
- [ ✔️ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

issues No:- #4438 (Minimize Amplitude After K Removals.py)

### CODE REVIEW

**Comment by kushal13112001** [comment]
sir @HarshCasper , @siddharth25pandey, @bhav09 
I have made the PR,
my file can be tested

**Comment by kushal13112001** [comment]
sir @HarshCasper , @siddharth25pandey, @bhav09, @ArchanGhosh 
I have made the PR,
my file can be tested

File: Python/cp/Minimize_Amplitude_After_K_Removals.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# k < n
+
+# Approach :- we have to sort the list first and take a 'p' variable
+# in which it stores the length of the list after removing the k.
+# Taken an 'm' variable which stores the min possible max(nums)-min(nums)
+# firstly and needs to be changed when we get the less max(nums)-min(nums).
+# Now, take a for loop which is to be iterated from 0 to len(nums)-p+1 ,
+# during the for loop I will check the difference i+p-1'th term to the i'th term
+# when I get the lease min I will change the min value to it.
+# At the end will return the 'm'(min(max(nums)-min(nums)))
```
Comment (ankitaggarwal23) [inline Python/cp/Minimize_Amplitude_After_K_Removals.py]: Use Multi-Line Comments

File: Python/cp/Minimize_Amplitude_After_K_Removals.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+# ST-2:- Input:-
+
+# nums = [84, 45, 2, 1, 894, 156, 68, 48]
+# k = 4
+
+# output:-
+# 39
+
+# Time Complexity:  O(nlogn)
+# Space Complexity: O(1)
```
Comment (ankitaggarwal23) [inline Python/cp/Minimize_Amplitude_After_K_Removals.py]: Use Multi-Line Comments
Remove the extra test case one is enough

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Few Changes:
- Use Multiline comments for description
- Remove redundant comments 
- Shift your code to some function and use main for driver code
- Write proper input output with display message

**Comment by kushal13112001** [comment]
Yes sir will change those @ankitaggarwal23

File: Python/cp/Minimize_Amplitude_After_K_Removals.py (hunk 2 above)
Comment (kushal13112001) [reply Python/cp/Minimize_Amplitude_After_K_Removals.py]: sir done @ankitaggarwal23

File: Python/cp/Minimize_Amplitude_After_K_Removals.py (hunk 1 above)
Comment (kushal13112001) [reply Python/cp/Minimize_Amplitude_After_K_Removals.py]: sir Done @ankitaggarwal23

**Comment by kushal13112001** [comment]
Yes sir done those changes @ankitaggarwal23 please review

**Comment by kushal13112001** [comment]
sir @HarshCasper , @siddharth25pandey, @bhav09, @ArchanGhosh
I have made the PR after doing the required changes
my file can be tested

**Review by HarshCasper — APPROVED** [review]

**Comment by kushal13112001** [comment]
tq @HarshCasper @ankitaggarwal23 sir for motivating me, I will work hard to give you the best questions for the repo

**Comment by HarshCasper** [comment]
Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs

It would be much appreciated. Thanks!

**Pull request merged**